src/pages/About.tsx

```diff
@@ -108,23 +58,78 @@ const About = () => (
           surname="Szolnoky"
         />
         <Profile
-          bio="Hej! Mitt namn är Charles och jag ansvarar mest för design här på digitalungdom.se! Det är även jag som har ritat bilderna som ni ser på hemsidan."
+          bio="Yo! Mitt namn är Charles och jag håller huvudsakligen på med front-end och design. På min fritid gillar jag att programmera spel och måla."
           color="lightGreen"
           name="Charles"
           role="Vice ordförande"
           surname="Maddock"
         />
         <Profile
           bio="Tjena! Jag är 20 år gammal och studerar på Handelshögskolan i Stockholm! Jag har i nuläget pausat mina studier i Datateknik på KTH. Jag jobbar med front-end och är kassör."
-          color="#1e6ee8"
+          color="#e0555c"
           name="Douglas"
           role="Kassör"
           surname="Bengtsson"
         />
-        <Profile bio="" color="#1e6ee8" name="Karl" role="Styrelseledamot" surname="Sellergren" />
+        <Profile
+          bio="Hej! Mitt namn är Karl och jag håller på med både front- och backend i Digital Ungdom. På min fritid gillar jag att segla!"
+          color="#6e73d4"
+          name="Karl"
+          role="Styrelseledamot"
+          surname="Sellergren"
+        />
       </Grid>
     </CenterWrapper>
   </div>
 );
 
+const Emoji = (props: any) => {
+  return (
+    <span aria-label="emoji" role="img">
+      {props.emoji}
+    </span>
+  );
+};
+
+interface ProfileProps {
+  name: string;
+  surname: string;
+  bio: string;
+  role: string;
+  color: string;
+}
+
+const Profile = ({ name, surname, bio, role, color }: ProfileProps) => (
+  <Grid item style={{ width: 260, margin: '0 8px' }}>
+    <div style={{ textAlign: 'center', width: '100%' }}>
+      <div
+        style={{
+          width: 260,
+          height: 260,
+          margin: '0 auto',
+          borderRadius: 10,
+          backgroundColor: color,
+          overflow: 'hidden',
+        }}
+      >
+        <img alt={name} src={require('resources/images/portraits/' + name.toLowerCase() + '.png')} />
+      </div>
+      <Typography style={{ fontSize: 20, fontWeight: 'bold', marginBottom: 2, marginTop: 10 }}>
+        {name + ' ' + surname}
+      </Typography>
+      <Typography style={{ fontSize: 16, fontStyle: 'italic', marginBottom: 16 }}>{role}</Typography>
+      <Typography style={{ fontSize: 14, marginBottom: 6, textAlign: 'left' }}>{bio}</Typography>
+
+      <Typography style={{ fontSize: 12, textAlign: 'justify', marginBottom: 24 }}>
```

Comment:
There are many examples where the code uses pixels to set font sizes. This is bad practice! Since this was implemented before this pull request, I'll let it pass and set up an issue to ask for a replacement. Nonetheless, pixels are bad.

You can read many articles on it, such as https://medium.com/@julienetienne/pixels-are-dead-faa87cd8c8b9

One of many reasons is that it does not scale with larger screens. Just because I have a 4k monitor does not mean that I should have read text that scales with the eyes of ants ;)

The solution is to use `rem` or `em`, and I suggest that usage of the theme / style provider we currently use (which is Material UI). [Here are the sizes they choose](https://material-ui.com/customization/default-theme/#default-theme) for subtitles, captions, bodies, titles etc. To implement this in the components one would either change the `variant` prop of a `Typography`-component. For example,

```tsx
<Typography variant="subtitle1">Douglas Bengtsson</Typography>
```

Reply:
Cool, I will try to fix this now

Reply:
Thanks for teaching me this :)